tests/mir-opt/inline/cycle.rs

```diff
@@ -1,20 +1,27 @@
-// skip-filecheck
 // EMIT_MIR_FOR_EACH_PANIC_STRATEGY
 // compile-flags: -Zinline-mir-hint-threshold=1000
 
 // EMIT_MIR cycle.f.Inline.diff
 #[inline(always)]
 fn f(g: impl Fn()) {
+    // CHECK-LABEL: fn f(
+    // CHECK-NOT: inlined
     g();
 }
 
 // EMIT_MIR cycle.g.Inline.diff
 #[inline(always)]
 fn g() {
+    // CHECK-LABEL: fn g(
+    // CHECK: (inlined f::<fn() {main}>)
+    // CHECK-NOT: inlined
```

Comment:
```suggestion
    // CHECK-NOT: inlined
    // CHECK: (inlined f::<fn() {main}>)
    // CHECK-NOT: inlined
```
This better ensures that `f` is the only inlined fn.